src/modules/auth/components/CustomDataContainer.js

```diff
@@ -73,8 +85,12 @@ export function CustomDataContainer({app, resources, customData, setCustomData,
 	const onSave = async (data) => {
 		// first, transform data to appropirate format for api request and skip pairs with empty key field
 		let objToSubmit = await prepForRequest(data);
+		// second, check if uri is in the right format to prevend invalid address
+		let address;
+		uri.charAt(0) === '/' ? address = uri.substring(1, uri.length)
```

Comment:
@petrKavulok You dont need to have this construction of conditions here at all, you can use regex expression and do simply:

```
let address = url.replace(/^\//g, "");
```

This will always remove the first trailing slash if it will appear in the `url` string. If there will be no trailing slash, the `url` string will stay intact.

You can also make it even simplier by using this in the request (then you dont need to declare `address` variable)

```
let response = await SeaCatAuthAPI.put(`/${url.replace(/^\//g, "")}`, {"data": objToSubmit});
```